File: MyCourses - Android/app/src/main/java/com/mycoursesapp/model/PromoCode.java
package com.mycoursesapp.model;

/**
 * Created by mamdouhelnakeeb on 2/14/18.
 */

public class PromoCode {

    public int id;
    public int discount;
    public String code;

    public PromoCode(int discount, String code){
        this.discount = discount;
        this.code = code;
    }

    public PromoCode(int discount, String code, int id){
        this.discount = discount;
        this.code = code;
        this.id = id;
    }

    public PromoCode(){}
}

File: MyCourses - iOS/My Courses/Models/PromoCode.swift
//
//  PromoCode.swift
//  My Courses
//
//  Created by Mamdouh El Nakeeb on 4/2/18.
//  Copyright © 2018 My Courses. All rights reserved.
//

import Foundation

class PromoCode {
    
    
    var id = 0
    var discount = 0
    var code = ""
    
    init(discount: Int, code: String){
        self.discount = discount
        self.code = code
    }
    
    init (discount: Int, code: String, id: Int){
        self.discount = discount;
        self.code = code;
        self.id = id;
    }
    
    init(){}
    
}
